CRM/Core/JobManager.php

```diff
@@ -24,6 +26,7 @@ class CRM_Core_JobManager {
    * Format is ($id => CRM_Core_ScheduledJob).
    *
    * @var CRM_Core_ScheduledJob[]
+   * @deprecated
```

Comment:
Adding "deprecated" and then removing the code is not ideal here.  Instead, the deprecated code should not immediately removed from the codebase because doing so may cause regression errors